pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1444,6 +1450,7 @@ type buildListenerOpts struct {
 	filterChainOpts []*filterChainOpts
 	bindToPort      bool
 	skipUserFilters bool
+	trafficDirection model.TrafficDirection
```

Comment:
@rshriram This field is not used in PR. Added the line here only to reflect the my question if the builder need to know the direction from here instead of from buildOutbound/buildInbound function.

Reply:
will remove